src/main/java/codeu/model/data/User.java

```diff
@@ -23,20 +23,22 @@
   private final String name;
   private final String hashedPassword;
   private final Instant creation;
+  private boolean isAdmin;
 
   /**
    * Constructs a new User.
    *
    * @param id the ID of this User
    * @param name the username of this User
-   * @param password the password of this User
+   * @param hashedPassword the users password
    * @param creation the creation time of this User
    */
   public User(UUID id, String name, String hashedPassword, Instant creation) {
     this.id = id;
     this.name = name;
     this.creation = creation;
     this.hashedPassword = hashedPassword;
+    isAdmin = true;
```

Comment:
Do we want to default all users as an Admin?

Reply:
That was a mistype on my end I'll fix that quick